content/blog/2016/2016-12-31-what-a-year.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Thank you for an amazing 2016
+tags:
+- jam
+- jenkins2
+- pipeline
+- blueocean
+- azure
+- gsoc
+author: rtyler
+---
+
+
+image:/images/jenkins-happy-new-year.png["Happy New Year from Jenkins", role=right]
+
+I do not think it is an exaggeration to say: 2016 was the best year yet for the
+Jenkins project. Since the first commit in 2006, the project has reached a
+number of significant milestones in its ten years but we have never experienced
+the breadth of major milestones in such a short amount of time. From
+link:/2.0/[Jenkins 2]
+and
+link:/projects/blueocean[Blue Ocean]
+to the
+link:/projects/gsoc[Google Summer of Code]
+and
+link:http://jenkinsworld.com[Jenkins World],
+
+I wanted to take a moment and celebrate the myriad of accomplishments which
+couldn't have happened without the help from everybody who participates in the
+Jenkins project. The *1,300+* contributors to the
+link:https://github.com/jenkinsci[jenkinsci GitHub organization],
+the *4,000+* members of the
+link:https://groups.google.com/forum/#!forum/jenkinsci-dev[developers mailing list],
+the *8,000+* members of the
+link:https://groups.google.com/forum/#!forum/jenkinsci-users[users mailing list],
+and countless others who have reported issues, submitted pull requests, and
+presented at meetups and conferences.
+
+
+== Jenkins 2
+
+Through the course of 2016, the Jenkins project published 16
+link:/changelog-stable[LTS releases]
+and 54
+link:/changelog[Weekly releases].
+Of those 70 releases, the most notable may have been the
+link:/blog/2016/04/26/jenkins-20-is-here/[Jenkins 2.0 release]
+which was published in April.
+
+Jenkins 2 made Pipeline as Code front-and-center in the user experience,
+introduced a new "Getting Started" experience, and included a number of other
+small UI improvements, all while maintaining backwards compatibility with
+existing Jenkins environments.
+
+Since April, we have released a number of LTS
+releases using Jenkins 2 as a baseline, meaning the Jenkins project no longer
+maintains _any_ 1.x release lines.
+
+The
+link:/doc/book/pipeline[Pipeline]
+efforts have continuted to gain steam since April, covered on this blog with a
+number of
+link:/node/tags/pipeline/[posts tagged "pipeline"]. Closing out 2016 with the
+announcement of the beta for
+link:/blog/2016/12/19/declarative-pipeline-beta/[Declarative Pipeline syntax]
+which is expected in early 2017.
+
+== Blue Ocean
+
+Hot on the heels of Jenkins 2 announcement
+link:/projects/blueocean["Blue Ocean, a new user experience for Jenkins"],
+was
+link:/blog/2016/05/26/introducing-blue-ocean/[open sourced in May].
+Blue Ocean is a new project that rethinks the user experience of Jenkins.
+Designed from the ground up for Jenkins Pipeline and compatible with Freestyle
+jobs, The goal for the project is to reduce clutter and increase clarity for
+every member of the team.
```

Comment:
"for every of the team"?  Which team?   

how about "for anyone who uses Jenkins"?